sp_HumanEvents/sp_HumanEventsBlockViewer.sql

```diff
@@ -545,12 +547,15 @@ SELECT
     clientoption1 = bd.value('(process/@clientoption1)[1]', 'bigint'),
     clientoption2 = bd.value('(process/@clientoption1)[1]', 'bigint'),
     currentdbname = bd.value('(process/@currentdbname)[1]', 'nvarchar(128)'),
+    blocking_level = 0,
+    sort_order = cast('' as varchar(max)),
```

Comment:
`sort_order` will indicate the blocking chain to the lead blocker. The chain is the sorting we want to use to make the blocking tree look sane.